### PR TITLE
Invoke and Compel

### PR DESCRIPTION
**Pull request by agude**
Writing up a bit on invoking and compelling.

### CODE REVIEW

File: _chapters/00-outline.md [hunk 1]
```diff
@@ -319,26 +319,45 @@ The details chosen should make sense for the story and progress the narrative.
...
+Fate points can be spent to:
 
-Fate points are earned by accepting compels.
-- Complication that makes sense in light of an aspect
-- Typically suggested by the GM
-- Often (but not always) related to a PC's trouble
+- **Invoke an aspect**: 
+  Add +2 to a roll if there is a relevant aspect that would help.
+  An aspect can only be invoked once per roll, but multiple aspects can be invoked on the same roll if the player has multiple fate points to spend.
+- **Create an aspect**:
```
Comment (charles-uno) [inline _chapters/00-outline.md:331]: for a while we called this compelling the GM. how does that feel?

File: _chapters/00-outline.md [hunk 2]
```diff
@@ -319,26 +319,45 @@ The details chosen should make sense for the story and progress the narrative.
...
-- Compelling the GM
-- Often related to a PC's past
-- I've been here before
-- That's my old boat
+```
+TODO:
+Is it always +2? Reroll?
+```
+
+Fate points are earned through accepting compels.
```
Comment (charles-uno) [inline _chapters/00-outline.md:341]: Would it make sense to have compels be their own section? I don't think there's a chicken/egg issue here where we have to define them together

File: _chapters/00-outline.md [hunk 3]
```diff
@@ -319,26 +319,45 @@ The details chosen should make sense for the story and progress the narrative.
...
+The GM compels a player by offering a fate points to forgo a roll and instead let the GM decide the result.
```
Comment (charles-uno) [inline _chapters/00-outline.md:342]: some mixed up plurals in this paragraph

File: _chapters/00-outline.md [hunk 4]
```diff
@@ -319,26 +319,45 @@ The details chosen should make sense for the story and progress the narrative.
...
+Players can suggest a compel to the GM and are free to negotiate any compel offered.
+A compel can always be refused by the player.
+
+A player might invoke or create an aspect in a way that harms another player's character.
+In this case, the GM should give the spent fate point to that player to compensate them.
+Treat this like a compel: you need the harmed player to accept.
+If they refuse, figure out a different course of action.
 
+Players begin each session with fate points equal to their PC's level.
+The GM has an unlimited pool of fate points for compels.
```
Comment (charles-uno) [inline _chapters/00-outline.md:353]: we say earlier that the GM starts with a pool of fate points equal to PC level. Are we changing that? Or do you mean that compels come from the bank, not from the GM's pile?

File: _chapters/00-outline.md (hunk 3 above)
Comment (charles-uno) [inline _chapters/00-outline.md:342]: forgo -> forfeit? Compels are explicitly "bad" for the PC

**Review by charles-uno — APPROVED** [review]

File: _chapters/00-outline.md (hunk 2 above)
Comment (agude) [reply _chapters/00-outline.md:336]: I think they fit naturally together. I'll add in a "sub-section" for compels but we can reorder a bit.

File: _chapters/00-outline.md (hunk 4 above)
Comment (agude) [reply _chapters/00-outline.md:348]: We actually haven't even written the section about opposed rolls. :-/ 

I think compels come from the bank. They're good, we want them.

File: _chapters/00-outline.md (hunk 1 above)
Comment (agude) [reply _chapters/00-outline.md:331]: I don't like it :-/

**Pull request merged**